src/Compute/custom/NewAzVmss_SimpleParameterSet.cs

```diff
@@ -0,0 +1,373 @@
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string PublicIpAddressName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string DomainNameLabel { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SecurityGroupName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string LoadBalancerName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] BackendPort { get; set; } = new[] { 80 };
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        //[LocationCompleter("Microsoft.Compute/virtualMachineScaleSets")]
+        public string Location { get; set; }
+
+        // this corresponds to VmSku in the Azure CLI
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string VmSize { get; set; } = "Standard_DS1_v2";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public UpgradeMode UpgradePolicyMode { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        [ValidateSet("Static", "Dynamic")]
+        public string AllocationMethod { get; set; } = "Static";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string VnetAddressPrefix { get; set; } = "192.168.0.0/16";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SubnetAddressPrefix { get; set; } = "192.168.1.0/24";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string FrontendPoolName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string BackendPoolName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage = "Use this to add system assigned identity (MSI) to the vm")]
+        public SwitchParameter SystemAssignedIdentity { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage = "Use this to add the assign user specified identity (MSI) to the VM")]
+        [ValidateNotNullOrEmpty]
+        public string UserAssignedIdentity { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public SwitchParameter EnableUltraSSD { get; set; }
+
+        [Parameter(
+            ParameterSetName = SimpleParameterSet,
+            Mandatory = false,
+            HelpMessage = "A list of availability zones denoting the IP allocated for the resource needs to come from.",
+            ValueFromPipelineByPropertyName = true)]
+        public List<string> Zone { get; set; }
```

Comment:
@markcowl should we change this type to `string[]`? This will resolve issues we have with docs generation, but I don't know if we still have those issues when using Yanni's generator.